README.md

```diff
@@ -1,18 +1,21 @@
-# ![Bot Framework SDK](./docs/media/BotFrameworkSDK.png)
+# ![Bot Framework](./docs/media/BotFrameworkSDK.png)
 
-### [Click here to find out what's new with Bot Framework](https://docs.microsoft.com/en-us/azure/bot-service/what-is-new?view=azure-bot-service-4.0)
+### [What's new with Bot Framework?](https://docs.microsoft.com/en-us/azure/bot-service/what-is-new?view=azure-bot-service-4.0)
 
-# Bot Framework SDK
+Bot Framework provides the most comprehensive experience for building conversation applications.  
 
-The Bot Framework SDK v4, part of the [Bot Framework](https://github.com/microsoft/botframework-sdk), provides the most comprehensive experience for building conversation applications. With the Bot Framework SDK, developers can easily model and build sophisticated conversation using their favorite programming languages. With the Bot Framework SDK, you can build bots that converse free-form or your bot can also have more guided interactions where it provides the user choices or possible actions. The conversation can use simple text or more complex rich cards that contain text, images, and action buttons. You can add natural language interactions and questions and answers, which let your users interact with your bots in a natural way.
+With the [Bot Framework SDK](#bot-framework-sdk-v4), developers can build bots that converse free-form or with guided interactions including using simple text or rich cards that contain text, images, and action buttons.  
 
-Checkout the [Bot Framework ecosystem](#bot-framework-ecosystem) section to learn more about other tooling and services related to the Bot Framework SDK.
+Developers can model and build sophisticated conversation using their favorite programming languages including C#, JS, and Python or using [Bot Framework Composer](https://aka.ms/bfcomposer), an open-source, visual authoring canvas for developers and multi-disciplinary teams to design and build conversational experiences with Language Understanding, QnA Maker and sophisticated composition of bot replies (Language Generation). 
```

Comment:
it feels a bit odd that we don't mention Java here.  It's a balance.  Java is in preview.  but, you can build bots with the Java SDK.  And later on in the quick links section we have a link to Java.  I think we should mention Java in this paragraph

Reply:
Java added.